Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.py

```diff
@@ -152,32 +148,25 @@ def ip_reputation_command(client: Client, args: Dict[str, Any], api_key) -> Comm
             dbot_score=dbot_score
         )
 
-        ip_standard_list.append(ip_standard_context)
-
-        ip_context_excluded_fields = ['objects', 'nir']
-        ip_data_list.append({k: ip_data[k] for k in ip_data if k not in ip_context_excluded_fields})
+        ip_data.pop('objects')
+        ip_data.pop('nir')
+        command_results.append(CommandResults(
+            readable_output=tableToMarkdown('IP List', ip_data),
```

Comment:
Given a single IP, I feel that IP List is not a good human-readable title, perhaps IP Details?